src/main/java/edu/ie3/datamodel/io/processor/ProcessorProvider.java

```diff
@@ -143,8 +251,29 @@ public ProcessorProvider(Collection<EntityProcessor<? extends UniqueEntity>> pro
     return resultingProcessors;
   }
 
+  /**
+   * Create processors for all known eligible combinations and map them
+   *
```

Comment:
I think I understand why this mapping is needed (guess also had a discussion) but why isn't it needed for entity processors? easier to ask: what is the difference between time series and entity processors?